sample-parent/sample-connector/src/main/resources/org/talend/sdk/component/test/connectors/Messages_en.properties

```diff
@@ -24,3 +24,5 @@ the_family.dataset.TheDataset._displayName = Name: Sample dataset -en
 the_family.actions.suggestions.action_LIST_ENTITIES._displayName = Name: LIST ENTITIES -en
 the_family.actions.update.action_UPDATE._displayName = Name: Update the nested config elements -en
 the_family.actions.validation.action_VALIDATION._displayName = Name: Validate the configuration -en
+the_family.actions.healthcheck.action_HEALTH_CHECK_OK._displayName = Name: Check the HealthCheck status OK -en
+the_family.actions.healthcheck.action_HEALTH_CHECK_KO._displayName = Name: Check the HealthCheck status KO -en
```

Comment:
Can you check if the result is the same with python script?

Reply:
The only difference it that missing Line break symbol.  Already use the python script file to update the messages properties.